x/evm/keeper/statedb_test.go

```diff
@@ -476,9 +475,43 @@ func (suite *KeeperTestSuite) TestEmpty() {
 }
 
 func (suite *KeeperTestSuite) TestSnapshot() {
```

Comment:
test looks good, I'd just clean it up and add a table test with the following cases

- **simple revert**: `Snapshot` and `RevertToSnapshot()`. Stack should be empty afterwards
- **sequential revert**: `rev1 := Snapshot`, `rev2 := Snapshot` and then  `RevertToSnapshot(rev2)` and `RevertToSnapshot(rev1)`. Stack should be empty afterwards
- **"jump" revert**:  `rev1 := Snapshot`, `rev2 := Snapshot` and then `RevertToSnapshot(rev1)`. Stack should be empty afterwards

Reply:
done